mitiq/benchmarks/randomized_benchmarking.py

```diff
@@ -62,6 +79,19 @@ def generate_rb_circuits(
     cfd_matrices = _two_qubit_clifford_matrices(
         *qubits, cliffords,  # type: ignore
     )
+    if return_type is not None:
+        return [
+            convert_from_mitiq(
+                _random_two_q_clifford(
+                    *qubits,  # type: ignore
+                    num_cliffords,
+                    cfd_matrices,
+                    cliffords,
+                ),
+                return_type,
+            )
+            for _ in range(trials)
+        ]
```

Comment:
I think it's (marginally) better to do something like below for readability / fewer LOC.

```python
circuits = [
        _random_two_q_clifford(
            *qubits,  # type: ignore
            num_cliffords,
            cfd_matrices,
            cliffords,
        )
        for _ in range(trials)
    ]

if return_type is None or return_type == "cirq":
    return circuits

return [convert_from_mitiq(circuit, return_type) for circuit in circuits]
```

Reply:
It's much better 😅  Can define the `random_single_q`s as `circuits` too then there's only one conditional return and one call to convert 💯